### DISCUSSION TITLE
Convert from .svg and .png files to .docx and pdf files

### DISCUSSION THREAD

**Discussion by QuangLanh** [clientIO/joint#2398] [Q&A] [answered]
### Introduction

Currently, the library I bought from you has the ability to export drawings to .svg and png files.
But I want to change it to .docx and .pdf files, is that possible?
If I can change it, where in the code should I edit it?

### Steps to reproduce

![image](https://github.com/clientIO/joint/assets/91791283/ee44384f-0f82-435f-ab39-5b8f3f02cf2b)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment, marked as answer]
_JointJS_ does not have any plugin to export the diagram to **PDF**, nor to **DOCX**.
What you need is to export the diagram to _PNG_/_SVG_ and add the image to the _PDF_/_DOCX_ document using some third party library.

Here's an example of export to _PDF_ using  _jspdf_:
https://www.jointjs.com/demos/pdf-export

Here is an example of a library for generating _DOCX_ files that provides the API you need:
https://github.com/dolanmiu/docx

**Reply by QuangLanh** [reply]
thank you so much!
